web/src/rendered_markdown.js

```diff
@@ -78,6 +79,13 @@ export const update_elements = ($content) => {
         $content.addClass("rtl");
     }
 
+    if (util.is_client_safari()) {
+        $content.find(".message_inline_video video").each(function () {
+            // Without this video thumbnail doesn't load on Safari.
```

Comment:
I would put this comment as the very first thing inside the `if` statement -- that's usually a very good place for comments explaining the purpose of a block. And then have it say something like "On Safari, one needs to manually load video elements" and also include a link to that MDN page about `load` just for clarity.